tests/cppunit/types/bitmap_test.cc

```diff
@@ -115,15 +132,43 @@ TEST_P(RedisBitmapTest, BitCountNegative) {
     EXPECT_FALSE(bit);
   }
 
-  bitmap_->BitCount(key_, 0, 1024, &cnt);
+  bitmap_->BitCount(key_, 0, 1024, false, &cnt);
   EXPECT_EQ(cnt, 4);
 
-  bitmap_->BitCount(key_, 0, 1023, &cnt);
+  bitmap_->BitCount(key_, 0, 1023, false, &cnt);
   EXPECT_EQ(cnt, 3);
 
   auto s = bitmap_->Del(key_);
 }
 
+TEST_P(RedisBitmapTest, BitCountBITOption) {
+  uint32_t offsets[] = {0, 100, 1024 * 8, 1024 * 8 + 1, 3 * 1024 * 8, 3 * 1024 * 8 + 1};
+  for (const auto &offset : offsets) {
+    bool bit = false;
+    bitmap_->SetBit(key_, offset, true, &bit);
+  }
```

Comment:
Nit: we can add more test case to just get self. ( `BitCount(key_, offset, offset, ..)` )

Reply:
You can add something like:

```
for (uint32_t bit_offset = 0; bit_offset <= 3 * 1024 * 8 + 1; ++bit_offset) {
  bitmap_->BitCount(key_, offset, offset, /*...*/true, &positive_cnt);
  bitmap_->BitCount(key_, offset, offset,  /*...*/false, &negative_cnt);
  // check the result.
}
```

Reply:
```c++
  for (uint32_t offset = 0; offset <= 3 * 1024 * 8 + 1; ++offset) {
    uint32_t cnt = 0;
    EXPECT_TRUE(bitmap_->BitCount(key_, offset, offset, /*bit=*/true, &cnt).ok());
    if (offset == 0 || offset == 100 || offset == 1024 * 8 || offset == 1024 * 8 + 1 || offset == 3 * 1024 * 8 ||
        offset == 3 * 1024 * 8 + 1) {
      EXPECT_EQ(1, cnt) << "offset: " << offset;
    } else {
      EXPECT_EQ(0, cnt) << "offset: " << offset;
    }
  }
```

E.g. Add the code above